schemas/vertices/gtdb/gtdb_vertices.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "description": "A entry for vertices in the GTDB taxonomy tree.",
+  "required": ["_key", "release", "type", "name"],
+  "optional": ["domain", "phylum", "class", "order", "family", "genus", "species"]
+  "properties": {
+    "_key": {
+      "type": "string",
+      "description": "Taxon type abbreviation plus name",
+      "examples": ["d:bacteria", "s:['salmonella', 'enterica']"]
```

Comment:
for the multiword, we should just replace spaces with underscores rather than having some kind of stringified array. Arangodb won't allow spaces ion the _key field.